configs/common/packages.yaml

```diff
@@ -168,7 +168,9 @@
       version: [3.10.5]
     nlohmann-json-schema-validator:
       version: [2.1.0]
-    # Do not build pkg-config - https://github.com/NOAA-EMC/spack-stack/issues/123
+    # Do not build pkgconf/pkg-config - https://github.com/NOAA-EMC/spack-stack/issues/123
+    pkgconf:
+      buildable: False
     pkg-config:
       buildable: False
```

Comment:
One of them should be buildable. An external `pkg-config` is generally available on these HPC systems. So that's why I say go for that.